src/se/terhol/pisemka22/ReservationException.java

```diff
@@ -0,0 +1,13 @@
+package se.terhol.pisemka22;
+
+/**
+ * @author Tereza Holm
+ */
+public class ReservationException extends Exception {
+    public ReservationException () {
+
+    }
+
+    public ReservationException(String message){
```

Comment:
What does this method? You probably miss some attribute assigment/super here.

Reply:
Done